package.json

```diff
@@ -29,6 +29,7 @@
   "dependencies": {
     "@carbon/icon-helpers": "^10.37.0",
     "@carbon/icons": "^11.14.0",
+    "@lit/context": "^1.1.0",
```

Comment:
This is neat, I didn't even know it was a thing. Could be very useful for component > sub-component communication.